workspaces/rbac/plugins/rbac/package.json

```diff
@@ -46,11 +46,9 @@
     "@backstage/plugin-permission-react": "^0.4.27",
     "@backstage/theme": "^0.6.0",
     "@janus-idp/shared-react": "^2.13.1",
-    "@material-ui/core": "^4.9.13",
-    "@material-ui/icons": "^4.11.3",
-    "@material-ui/lab": "^4.0.0-alpha.45",
     "@mui/icons-material": "5.16.4",
     "@mui/material": "^5.14.18",
+    "@mui/styles": "^6.1.7",
```

Comment:
6.1.7? Ain't this suppose to be 5?

Reply:
@drodil Thanks for reviewing it . "^6.1.7" is the latest version of @mui/styles which is compatible with MUI v5 . Let us know the reason if you still think we should have used version 5